apps/web/src/stores/documentVersions.ts

```diff
@@ -150,11 +159,33 @@ export default function useDocumentVersions(
         }
       }
     },
-    [executeDestroyFolder, mutate, currentDocument?.path],
+    [executeDestroyFolder, mutate, currentDocument?.path, commit.id],
   )
 
-  const { execute: updateContent } = useLatitudeAction(
```

Comment:
Why this? We should use `useLatitudeAction` to have a unique way of doing actions

Reply:
Every other action in here uses the regular `useServerAction`